website/content/en/docs/kubebuilder-migration/project_migration_guide.md

```diff
@@ -0,0 +1,275 @@
+# Project Migration - Single API version operator
+
+This guide walks through the steps required to migrate an operator from an Operator SDK project layout to a Kubebuilder project layout.
+
+The document considers [memcached operator][memcached-operator] as an example to describe the process of migrating an operator built using Operator SDK to a runnable Kubebuilder project.
```

Comment:
Is it valid for all types? If not then, I think it should be in the Golang section.

Reply:
@hasbro17 Are we keeping it in a separate folder as of now, or can we move it to golang section?

Reply:
Why we ned to add `Single API version operator`? What does it mean? 
what relation it has with migrate the layout of the projects?

Reply:
Since we are not explaining anything about Multi-Version API (not anytime soon I think), I've specifically mentioned Single API version operator.

Reply:
Hun, but add on top lead to me believe that this is the main topic of the doc. 
Also, why not add a note such as follows to cover the multi-group scenario? 

**Muti-Group**

If your project as multi groups for its API's then you will need to run the command `operator-sdk edit --multigroup=true` to allow this layout. 

Notice that with the multi-group project the Kind API’s files are created under apis/<group>/<version> instead of api/<version>. Also, note that the controllers will be created under controllers/<group> instead of controllers. That is the reason why we moved the previously generated APIs with the provided scripts in the previous steps. Remember to update the references afterwards.

For further information see: https://book.kubebuilder.io/migration/multi-group.html

Reply:
Since we do not support Webhooks implementation (as in https://book.kubebuilder.io/multiversion-tutorial/conversion.html) do we need to add it?

Reply:
@varshaprasad96 Multi-group is different from multi-version for a single API.
You can have 2 groups both with a single version each so no conversion webhooks required.

The problem for multi-groups right now is that you have to run `kubebuilder edit --multigroup=true` after adding the first group. But that command isn't available yet in the new operator-sdk CLI.
So I need to double check with @estroz if we'll have that command.

But to cover the multi-group scenario you can add the above note for enabling multi-groups if they have multiple groups, before adding any APIs. 
And instead of using the step `operator-sdk edit --multi-group=true` users will need to manually modify their PROJECT file to add the line `multigroup: true` so that the PROJECT file ends up looking like this:

```YAML
domain: example.com
layout: go.kubebuilder.io/v2.0.0
repo: github.com/hasbro17/memcached-operator
multigroup: true
version: 3-alpha
plugins:
  go.operator-sdk.io/v2.0.0: {}
```

Then if you run `create api` it will scaffold apis in `apis/<group>/<version>/` and controllers in `controllers/<group>/`.